gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/DiagnosticsManifestPlugin.java

```diff
@@ -95,39 +93,23 @@ public void apply(Project project) {
         });
     }
 
-    private static void configureExternalDependencyTransform(Project project) {
+    private static void configureJarTransform(Project project) {
         project.getDependencies().registerTransform(ExtractFileFromJar.class, details -> {
             details.getParameters().getPathToExtract().set(FILE_TO_EXTRACT);
 
             details.getFrom().attribute(artifactType, ArtifactTypeDefinition.JAR_TYPE);
-            details.getTo().attribute(artifactType, MY_ATTRIBUTE);
+            details.getTo().attribute(artifactType, DIAGNOSTICS);
 
+            // It's not strictly necessary to define a mapping for the 'org.gradle.libraryelements' attribute,
+            // but it looks a bit weird for something to still be marked as a 'jar' when it's clearly not.
```

Comment:
I was a bit confused why we set the `org.gradle.libraryelements` attribute, since it didn't seem necessary.

Then I discovered that @iamdanfox's had some good comments in https://github.com/palantir/sls-packaging/pull/1035 that were removed before the PR merged.

https://github.com/palantir/sls-packaging/blob/614f8a265085d3fcdf32e70c81b7c56ef4da9ac0/gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/DiagnosticsManifestPlugin.java#L95-L96

Adding this back so future devs understand why we do this.